pkg/ebpf/ksyms.go

```diff
@@ -47,15 +48,16 @@ func GetKernelSymbolsAddresses(ksyms ...string) (map[string]uint64, error) {
 	}
 
 	addresses := make(map[string]uint64, len(ksyms))
+	var errs []error
 	for _, sym := range ksyms {
 		if _, ok := missing[sym]; ok {
-			return nil, fmt.Errorf("failed to get address of symbol %s", sym)
+			errs = append(errs, fmt.Errorf("failed to get address of symbol %s", sym))
```

Comment:
nit: maybe we can spare the logs output and return a single error that prints the list of unverified symbols.